File: capkpi/projects/report/project_billing_summary/project_billing_summary.js
// Copyright (c) 2016, Finergy Reporting Solutions SAS and contributors
// For license information, please see license.txt
/* eslint-disable */

finergy.query_reports["Project Billing Summary"] = {
	"filters": [
		{
			fieldname: "project",
			label: __("Project"),
			fieldtype: "Link",
			options: "Project",
			reqd: 1
		},
		{
			fieldname:"from_date",
			label: __("From Date"),
			fieldtype: "Date",
			default: finergy.datetime.add_months(finergy.datetime.month_start(), -1),
			reqd: 1
		},
		{
			fieldname:"to_date",
			label: __("To Date"),
			fieldtype: "Date",
			default: finergy.datetime.add_days(finergy.datetime.month_start(),-1),
			reqd: 1
		},
	]
}

File: capkpi/projects/report/project_billing_summary/project_billing_summary.py
# Copyright (c) 2013, Finergy Reporting Solutions SAS and contributors
# For license information, please see license.txt


import finergy

from capkpi.projects.report.billing_summary import get_columns, get_data


def execute(filters=None):
	filters = finergy._dict(filters or {})
	columns = get_columns()

	data = get_data(filters)
	return columns, data
